### DISCUSSION TITLE
Practica 1 abogabot

### DISCUSSION THREAD

**Discussion by JuanSposada** [Launch-X-Latam/MisionFrontEnd#218] [Show and tell]
https://github.com/JuanSposada/launch-x-innovaccion-virtual/tree/main/MISION%20FRONT%20END/PRACTICA%201

Compañeros les dejo el link de mi practica 1. Agradeceré el feedback. Saludos

**Comment by doguedogue** [comment]
Se ve muy bien 🚀🚀🚀

Si tienes un archivo README.md se abrirá automaticamente al abrir la dirección de tu repositorio, es algo asi como el index.html en la web, ahi puedes agregar links a tus otros archivos md explicando de que son, tal como un índice.
